### ISSUE TITLE
QEditor component toolbar buttons lack accessible name/label

### ISSUE THREAD

**Issue by wreality** [quasarframework/quasar#13275]
### What happened?

The QEditor toolbar buttons, by default, do not appear to have an accessible name for screen readers.  

### What did you expect to happen?

An accessible name (aria-tag, etc) to be present on the default buttons.

### Reproduction URL

https://codepen.io/wreality/pen/LYewevZ

### How to reproduce?

1. Open code pen.
2. Inspect a toolbar button in Chrome
3. Chrome DevTools:->Accessibility sidebar tab, note aria-role="button", but no label, labeled-by, or contents is available

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

Components (quasar)

### Platforms/Browsers

Chrome

### Additional context

A *very* naïve proposed solution is here: https://github.com/wreality/quasar/commit/93d14ad5994b6519063ac94684dac905674c9efb
